docs/operator-manual/notifications/services/awssqs.md

```diff
@@ -63,7 +63,7 @@ stringData:
 
 ### Minimal configuration using AWS Env variables
 
-Ensure following list of enviromental variable is injected via OIDC, or other method. And assuming SQS is local to the account.
+Ensure following list of environmental variable is injected via OIDC, or other method. And assuming SQS is local to the account.
```

Comment:
Personally, "environment variable " sounds better to me.

Reply:
ah fully agree!

Reply:
Updated; in fact it also should have been made plural, so just improved the wording.  Thanks for the feedback @zeusal !